README.md

```diff
@@ -232,6 +232,28 @@ Full usage example:
 dbt-coves load airbyte --host http://airbyte-server --port 8001 --path /config/workspace/load --secrets-path /config/workspace/secrets
 ```
 
+## Run dbt commands
+
+``` shell
+dbt-coves dbt <arguments> "<command>"
+```
+
+Run dbt commands on an isolated/prepared environment, with the possibility of changing dbt project location and activating a specific virtual environment in which running commands.
```

Comment:
I'd mention here that this is specially useful when running dbt on airflow workers.